crates/uv/src/commands/pip/tree.rs

```diff
@@ -57,14 +57,12 @@ pub(crate) fn pip_tree(
     .join("\n");
     writeln!(printer.stdout(), "{rendered_tree}").unwrap();
     if rendered_tree.contains('*') {
-        writeln!(
-            printer.stdout(),
-            "{}",
+        let message = if no_dedupe {
+            "(*) Dependency cycle".italic()
```

Comment:
Should we say "Package tree is a cycle and cannot be shown" as we discussed before?

Reply:
Or "Package tree forms/contains a cycle and cannot be shown"